frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -209,10 +211,15 @@ class DashboardHeader extends Component {
       toggleSidebar,
       isShowingDashboardInfoSidebar,
       closeSidebar,
+      databases,
     } = this.props;
 
     const canEdit = dashboard.can_write && isEditable && !!dashboard;
 
+    const hasModelActionsEnabled = isDatabaseActionsEnabled(
+      Object.values(databases),
+    );
```

Comment:
I think we should rather have a utility checking the actions setting for _one_ database, and then do something like that:

```js
const hasModelActionsEnabled = Object.values(databases).some(isDatabaseActionsEnabled)
```

Otherwise we won't be able to use it when we have a single database